data-filter.html

```diff
@@ -1312,13 +1331,34 @@ <h3 style="text-align: center;">Selected Measurements</h3>
                     self._cardSelectAll();
                     return;
                 }
+                const groupedByAnnotation = {};
+                const annotationType = this._attributes[this._groupedBy];
+
+                if (annotationType) {
+                  data.reduce((acc, attr) => {
+                    //info: if data is incorect
+                    if (!attr[annotationType.name]) {
```

Comment:
pay attention at this changes. in some cases there is no certain annotation type

Reply:
Do you have a project that showed this behavior ?

Reply:
yes, hg19 "Microglia epigenetic data from Nott et al."
https://trello-attachments.s3.amazonaws.com/5fb2bf730c3ed8260bb88d40/601960f043d95a1425eb8461/7a749e5b30044aa0b0a77a7f70b43543/image.png

Reply:
that is an incorrect state, 

the chart type is "StackedBlocksTrack", but it shows all measurement from that collection ? It should've filtered the measurements by chart type.

Reply:
get it. I will find and fix it, thanks

Reply:
it works fine for me :)